src/main/java/randoop/reflection/AbstractOperationModel.java

```diff
@@ -0,0 +1,82 @@
+package randoop.reflection;
+
+import java.util.*;
+import randoop.contract.*;
+import randoop.generation.ComponentManager;
+import randoop.main.GenInputsAbstract;
+import randoop.operation.*;
+import randoop.sequence.Sequence;
+import randoop.test.ContractSet;
+import randoop.types.ClassOrInterfaceType;
+import randoop.types.Type;
+import randoop.util.MultiMap;
+
+/** Created by Michael on 3/3/2017. */
```

Comment:
Documentation added.

Ambiguities about this class are also in the other comment about code duplication in `WeightedConstantsOperationModel.java`.